web/src/osm2streets-svelte/import/Osm2streetsSettings.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+  export let settings = {
```

Comment:
Note we don't have to do anything to detect changes to the form and propagate. `bind:value` makes it a two-way to this prop here, and then since the caller does `bind:` as well, they can then use reactive statements and listen for changes. https://svelte.dev/tutorial/component-bindings